modules/op-release-notes-1-7.adoc

```diff
@@ -0,0 +1,307 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-7_{context}"]
+= Release notes for {pipelines-title} General Availability 1.7
+
+With this update, {pipelines-title} General Availability (GA) 1.7 is available on {product-title} 4.10.
+
+[id="new-features-1-7_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.7.
+
+[id="pipelines-new-features-1-7_{context}"]
+=== Pipelines
+
+* With this update, the `when` expressions in a `Task` object are scoped to guard the tasks by default. To continue guarding the `Task` object and its dependent tasks, set the `scope-when-expressions-to-task` flag to `true`. 
++
+[NOTE]
+====
+The `scope-when-expressions-to-task` flag will be removed in a future release. As a best practice for {pipelines-shortname}, use `when` expressions scoped to the guarded `Task` only.
+====
+// https://github.com/tektoncd/pipeline/pull/4580
+
+* With this update, you can use variable substitution in the `subPath` field of a workspace within a task.
+// https://github.com/tektoncd/pipeline/pull/4351
+
+* With this update, you can reference parameters and result with a bracket notation, using both single and double quotes. Prior to this update, you could only use the dot notation. For example, the following are now equivalent:
+** `$(param.myparam)`, `$(param['myparam'])`, and `$(param["myparam"])`. 
++
+Using the bracket notation, you can also work with parameter names containing conflicting characters, such as `"."`. For example, `$(param['my.param'])` and `$(param["my.param"])`.
```

Comment:
There's nothing wrong with the style or format of this phrase. However, I think the intention of this phrase is to communicate something more like this:
```suggestion
You can use single or double quotes to enclose parameter names that contain problematic characters, such as `"."`. For example, `$(param['my.param'])` and `$(param["my.param"])`.
```